From 3869a0294b5ad7d2d98d83f9c11c96c0396405d9 Mon Sep 17 00:00:00 2001
From: Kay Potthoff <Kay.Potthoff@microsys.de>
Date: Tue, 16 Mar 2021 14:29:53 +0100
Subject: [PATCH 11/38] dts: fsl-s32g274a: added SerDes clocks
Organization: MicroSys Electronics GmbH

Added missing clocks for SerDes0/1 lane 1.

Signed-off-by: Kay Potthoff <Kay.Potthoff@microsys.de>
---
 arch/arm/dts/fsl-s32g274a.dtsi | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/fsl-s32g274a.dtsi b/arch/arm/dts/fsl-s32g274a.dtsi
index 9aa8a28c51da..8ad40c6efa6f 100644
--- a/arch/arm/dts/fsl-s32g274a.dtsi
+++ b/arch/arm/dts/fsl-s32g274a.dtsi
@@ -146,6 +146,8 @@
 			/* PFE0 */
 			<&clks S32G274A_SCMI_CLK_PFE0_RX_SGMII>,
 			<&clks S32G274A_SCMI_CLK_PFE0_TX_SGMII>,
+            <&clks S32G274A_CLK_SERDES1_LANE1_CDR>,
+            <&clks S32G274A_CLK_SERDES1_LANE1_TX>,
 			<&clks S32G274A_SCMI_CLK_PFE0_RX_RGMII>,
 			<&clks S32G274A_SCMI_CLK_PFE0_TX_RGMII>,
 			/* PFE1 */
@@ -160,6 +162,7 @@
 			<&clks S32G274A_SCMI_CLK_PFE2_TX_RGMII>;
 		clock-names = "pe",
 			"mac0_rx_sgmii", "mac0_tx_sgmii",
+            "mac0_rx1_sgmii", "mac0_tx1_sgmii",
 			"mac0_rx_rgmii", "mac0_tx_rgmii",
 			"mac1_rx_sgmii", "mac1_tx_sgmii",
 			"mac1_rx_rgmii", "mac1_tx_rgmii",
@@ -327,10 +330,13 @@
 		 <&clks S32GEN1_SCMI_CLK_GMAC0_RX_RGMII>,
 		 <&clks S32GEN1_SCMI_CLK_GMAC0_TX_RGMII>,
 		 <&clks S32GEN1_SCMI_CLK_GMAC0_TS_RGMII>,
-		 <&clks S32GEN1_SCMI_CLK_GMAC0_AXI>;
+		 <&clks S32GEN1_SCMI_CLK_GMAC0_AXI>,
+		 <&clks S32G274A_CLK_SERDES0_LANE1_CDR>,
+         <&clks S32G274A_CLK_SERDES0_LANE1_TX>;
 	clock-names = "rx_sgmii", "tx_sgmii", "ts_sgmii",
 		      "rx_rgmii", "tx_rgmii", "ts_rgmii",
-		      "axi";
+		      "axi",
+		      "rx1_sgmii", "tx1_sgmii";
 	pinctrl-0 = <&pinctrl0_gmac0 &pinctrl0_gmac0_mdio>;
 	pinctrl-1 = <&pinctrl0_gmac0_mdio>;
 	pinctrl-names = "gmac_rgmii", "gmac_sgmii";
--
2.30.2

